apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx

```diff
@@ -41,37 +49,45 @@ const CtaSlide = ({
         zIndex={10}
       >
         {child && <Box mb={4}>{child}</Box>}
-        <Typography
-          fontSize={{
-            xs: 'clamp(36px, 8vw, 48px)',
-            md: 'clamp(48px, 10vw, 58px)',
-          }}
-          variant={'h1'}
-          color={textColor}
-        >
-          {title}
-        </Typography>
+        {fetchFromStrapi ? (
+          <BlockRendererClient content={subhead as BlocksContent} />
+        ) : (
+          <Typography
+            fontSize={{
+              xs: 'clamp(36px, 8vw, 48px)',
+              md: 'clamp(48px, 10vw, 58px)',
+            }}
+            variant={'h1'}
+            color={textColor}
+          >
+            {subhead as string}
+          </Typography>
+        )}
```

Comment:
Why this logic is inside the component?
The component should behave the same if the data are coming from Strapi or from static contents; the component should receive information, typed as props, and render them